persistence/dynamostore.py

```diff
@@ -16,7 +16,7 @@ def __init__(self, table_name: str):
         try:
             self.table.table_status in (
                 "CREATING", "UPDATING", "DELETING", "ACTIVE")
-        except ClientError:
+        except self.dynamo.meta.client.exceptions.ResourceNotFoundException:
```

Comment:
but if table is not created i get
> botocore.exceptions.ClientError: An error occurred (UnrecognizedClientException) when calling the CreateTable operation: The security token included in the request is invalid.

Reply:
sorry, but its works same as ClientError, idk why :)

Reply:
it's interesting. For me ClientError doesn't catch the issue and give me the answer: 
```
botocore.errorfactory.ResourceNotFoundException: An error occurred (ResourceNotFoundException) when calling the DescribeTable operation: Cannot do operations on a non-existent table
```

Reply:
But I can catch the error from ClientError, if I run my tests. It raises an exception to my token. It seems, there are two types of issues.

Reply:
ClientError issue belongs to the behavior of the tests (without predefined ENV it tries to get default AWS creds and raises client exception)